common-theme/layouts/_default/success.html

```diff
@@ -9,7 +9,7 @@
       {{/* print any objectives from the success page front matter */}}
       {{ with .Params.Objectives }}
         <section class="c-objectives">
-          <h3 class="c-objectives__title is-invisible">Overall Objectives</h3>
+          <h3 class="c-objectives__title">Objectives</h3>
```

Comment:
Please move your theme changes to a different PR so they can be considered separately. 

Your content changes for ITP shouldn't be batched in with your changes for all possible users of this theme.